api/src/pcapi/core/geography/models.py

```diff
@@ -22,6 +23,8 @@ class Address(PcObject, Base, Model):
     city: str = sa.Column(sa.Text(), nullable=False)
     latitude: float = sa.Column(sa.Numeric(8, 5), nullable=False)
     longitude: float = sa.Column(sa.Numeric(8, 5), nullable=False)
+    departementCode = sa.Column(sa.Text(), nullable=True, index=True)
```

Comment:
Je propose, pour cette nouvelle table, de corriger la faute qui existait dans `Venue`, car c'est lourd de changer un nom de colonne ensuite :
```suggestion
    departmentCode = sa.Column(sa.Text(), nullable=True, index=True)
```